src/pages/waiting-room/waiting-room.ts

```diff
@@ -35,12 +36,18 @@ export class WaitingRoomPage extends BasePageComponent {
     public navParams: NavParams,
     public platform: Platform,
     public authenticationProvider: AuthenticationProvider,
+    private deviceProvider: DeviceProvider,
   ) {
     super(platform, navCtrl, authenticationProvider);
   }
 
   ionViewDidEnter(): void {
     this.store$.dispatch(new waitingRoomActions.WaitingRoomViewDidEnter());
+    this.deviceProvider.enableSingleAppMode();
```

Comment:
Can you add some protection around it running so it only runs on ios devices as the plugin will not exist otherwise. A call to super.isIos() should be enough